code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1278,7 +1278,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		playsound(target, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
 		target.visible_message("<span class='danger'>[user] attempted to disarm [target]!</span>", \
 						"<span class='userdanger'>[user] attemped to disarm [target]!</span>", null, COMBAT_MESSAGE_RANGE)
-
+		add_logs(user, target, "disarmed", " attempting to disarm")
```

Comment:
Think the args are incorrect here, why not just `(user, target, "attempted to disarm")`?

Reply:
That's format the other log for shoving someone used. I'll fix them up